test/integration/targets/net_static_route/tests/eos/basic.yaml

```diff
@@ -0,0 +1,102 @@
+---
+- name: setup - remove config used in test
```

Comment:
These are same tests as above cli/basic.yaml?

Reply:
Yeah same test. but for platform agnostic `net_static_route`.